.github/workflows/preview-docs.yml

```
@@ -1,16 +1,15 @@
 name: Preview docs
 
 on:
-  pull_request_target:
+  # pull_request_target:
+  pull_request:
```

```suggestion
  pull_request_target:
```

A maintainer-only trick to use a secret on PRs.